docker/BUILD.bazel

```diff
@@ -13,9 +13,17 @@
 # limitations under the License.
 
 genrule(
-    name = "orchestration_image_tar",
-    outs = ["orchestration-image.tar"],
-    cmd = "$(location :orchestration_image_builder) -o $@",
+    name = "orchestration_image_with_local_source_tar",
```

Comment:
suffixes `local-source` and `apt-install` are implementation details, I suggest to use name reflecting how the images are meant to be used. Suggest names `orchestration_image` used for building the regular image using the published packages and using `orchestration_image_localdev` for the image using the packages built from local source.

Also, `apt-install` is misleading as both flows uses `apt install`.

Reply:
Done.